includes/class-wc-stripe-webhook-handler.php

```diff
@@ -452,6 +452,50 @@ public function process_webhook_refund( $notification ) {
 		}
 	}
 
+	/**
+	 * Process webhook reviews that are opened. i.e Radar.
+	 *
+	 * @since 4.0.6
+	 * @param object $notification
+	 */
+	public function process_review_opened( $notification ) {
+		$order = WC_Stripe_Helper::get_order_by_charge_id( $notification->data->object->charge );
+
+		if ( ! $order ) {
+			WC_Stripe_Logger::log( 'Could not find order via charge ID: ' . $notification->data->object->charge );
+			return;
+		}
+
+		if ( $notification->data->object->open ) {
```

Comment:
@roykho If you can add a filter so themes can override the default functionality and change the order status to `on-hold` or `processing`, that would be amazing. @WPprodigy agrees with this approach, just like in this closed PR https://github.com/woocommerce/woocommerce-gateway-stripe/pull/536

Reply:
If there is a third party ERP integration with Woo, then the orders would be synced periodically over to let's say NetSuite and the store owner would want to review the orders that are flagged by Stripe. And to prevent these orders being synced, the store owner may want to set the order status to `on-hold` so it won't be part of the sync to fulfillment.